roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -229,7 +229,7 @@
       dest: "{{ tempdir }}/templates/logging-es-pvc.yml"
     vars:
       obj_name: "{{ openshift_logging_elasticsearch_pvc_name }}"
-      size: "{{ openshift_logging_elasticsearch_pvc_size }}"
+      size: "{{ (openshift_logging_elasticsearch_pvc_size | trim | length == 0) | ternary('10Gi', openshift_logging_elasticsearch_pvc_size) }}"
```

Comment:
So i tried your suggestion and the result when I provided no inventory var was the same as initially reported by this issue.  The requested size resulted in '0' instead of the default